installer/macos/create_pkg.sh

```diff
@@ -7,9 +7,12 @@ set -e
 PKGVERSION=dev
 ARCH=$1
 
-MASSASTATION_INSTALLER_NAME=MassaStation.app
+MASSASTATION_APPLICATION_NAME=MassaStation.app
 MASSASTATION_BINARY_NAME=massastation
 
+APPLE_DEVELOPER_ID_APPLICATION=$2
+APPLE_DEVELOPER_ID_INSTALLER=$3
+
 HOMEBREW_INSTALL_SCRIPT_URL=https://raw.githubusercontent.com/massalabs/homebrew.sh/master/homebrew-3.3.sh
 
 LICENSE_FILE_NAME=MassaStation_ToS.rtf
```

Comment:
I can't comment on the `display_usage` function so I'm commenting here. 
I think we need to update it to add the App Dev ID and Installer Dev ID